### PR TITLE
App: Seperate pipeline pause and close

### PR DESCRIPTION
**Pull request by niley7464**
This patch seperates pipeline pause and close.
Also, now we can see pipeline status in main activity.

This PR needs https://github.com/nnstreamer/nnstreamer/pull/4488

### CODE REVIEW

**Comment by wooksong** [comment]
What do the error messages above mean? Is pause->close OK, but stop isn't?

**Comment by niley7464** [comment]
> What do the error messages above mean? Is pause->close OK, but stop isn't?

I left the explanation too abstractly.

I called `ml_pipeline_start`-> `ml_pipeline_stop` (pause) -> `ml_pipeline_start`

Then, restared server recived data like below.
```
2024-06-13 12:45:57.783 22590-22666 nnstreamer-edge         ai....eamer.ml.inference.offloading  D  Received command:0 (num:0)
2024-06-13 12:45:57.784 22590-22666 nnstreamer-edge         ai....eamer.ml.inference.offloading  E  Failed to get host info.
2024-06-13 12:45:57.784 22590-22666 nnstreamer-edge         ai....eamer.ml.inference.offloading  D  Send error cmd to close connection.
```

Data should have command enum and memory number like this.
```
2024-06-13 12:48:29.742 22884-22977 nnstreamer-edge         ai....eamer.ml.inference.offloading  D  Received command:2 (num:1)
2024-06-13 12:48:29.758 22884-22982 nnstreamer-edge         ai....eamer.ml.inference.offloading  D  Received command:1 (num:1)
```

So I thought the error message appeared when `ml_pipeline_close` called in PR description might be the cause.

**Review by wooksong — COMMENTED** [review]
We don't need to add submodule's changes.

File: ml_inference_offloading/src/main/res/layout/models.xml [hunk 1]
```diff
@@ -13,9 +13,15 @@
         android:id="@+id/stop"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Stop" />
+    android:text="Stop" />
```
[the remark this answers is not in the document]
Comment (niley7464) [reply ml_inference_offloading/src/main/res/layout/models.xml]: Updated
Thank you :)

**Comment by niley7464** [comment]
> We don't need to add submodule's changes.

Removed submoudle's new commit.
Thank you :)

**Review by jaeyun-jung — APPROVED** [review]

**Review by wooksong — APPROVED** [review]

**Pull request merged**